templates/devspace.yaml.tpl

```diff
@@ -157,6 +157,18 @@ dev:
         {{- if (has "node" (stencil.Arg "grpcClients")) }}
         - api/clients/node/node_modules/
         {{- end }}
+    {{- range (stencil.GetModuleHook "devspace.sync") }}
+    - name: {{ .name }}
```

Comment:
Doesn't this significantly muddy the move to devspace v2? I'm... a little worried about that. Though I guess we probably already have module hooks here.

Reply:
Yeah we have a few module hooks already and `stencil-smartstore` is using those as well. The new filesync syntax seems to be mostly compliant with this, so we may not even have to change the hooks themselves. That was why I used all these variables here.

New syntax docs: https://www.devspace.sh/docs/configuration/dev/connections/file-sync